### PR TITLE
add content roles

### PR DESCRIPTION
**Pull request by Thiblef**

### CODE REVIEW

File: BackofficeBundle/Resources/translations/role.en.yml [hunk 1]
```diff
@@ -5,8 +5,11 @@ open_orchestra_role:
     role_access_update_node: Update a page
     role_access_delete_node: Delete a page
     role_access_tree_template: Manage page templates
-    role_access_content_type_for_content: Manage contents
-    role_access_content_type: Manage content types
+    role_access_content_type_for_content: Manage a content
```
Comment (itkg-alavieille) [inline BackofficeBundle/Resources/translations/role.en.yml]: no update this line, use plurial

File: BackofficeBundle/Resources/translations/role.en.yml [hunk 2]
```diff
@@ -5,8 +5,11 @@ open_orchestra_role:
...
+    role_access_create_content_type_for_content: Create a content
+    role_access_update_content_type_for_content: Update a content
+    role_access_delete_content_type_for_content: Delete a content
+    role_access_content_type: Manage a content type
```
Comment (itkg-alavieille) [inline BackofficeBundle/Resources/translations/role.en.yml]: no update this line, use plurial

File: BackofficeBundle/Resources/translations/role.fr.yml [hunk 3]
```diff
@@ -5,8 +5,11 @@ open_orchestra_role:
     role_access_update_node: Editer une page
     role_access_delete_node: Supprimer une page
     role_access_tree_template: Gérer les gabarits de pages
-    role_access_content_type_for_content: Gérer les contenus
-    role_access_content_type: Gérer les types de contenu
+    role_access_content_type_for_content: Gérer un contenu
```
Comment (itkg-alavieille) [inline BackofficeBundle/Resources/translations/role.fr.yml]: no update this line, use plurial

**Comment by Thiblef** [comment]
@itkg-alavieille translations fixed

**Comment by itkg-alavieille** [comment]
+1::

**Pull request merged**